Gemfile.lock

```diff
@@ -42,20 +42,13 @@ GEM
       method_source (~> 0.8.1)
       slop (~> 3.4)
     rack (1.6.0)
-    rack-protection (1.5.3)
```

Comment:
To be honest I just ran bundle install. The changes seemed good ( like you said ) so I decided to push it.